### PR TITLE
Adding about_Aliases

### PR DESCRIPTION
**Pull request by sethvs**
This change is [Reviewable](https://reviewable.io/reviews/powershell/powershell-docs.ru-ru/6)

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md](https://github.com/sethvs/powerShell-Docs.ru-ru/blob/inprogress/reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Core/ToC.md](https://github.com/sethvs/powerShell-Docs.ru-ru/blob/inprogress/reference/5.1/Microsoft.PowerShell.Core/ToC.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cfca22aef-d511-d56e-fd47-4db9de768a0c%5CPullRequest%5C201706221819061612-6%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md](https://github.com/sethvs/powerShell-Docs.ru-ru/blob/inprogress/reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Core/ToC.md](https://github.com/sethvs/powerShell-Docs.ru-ru/blob/inprogress/reference/5.1/Microsoft.PowerShell.Core/ToC.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C23%5Cfca22aef-d511-d56e-fd47-4db9de768a0c%5CPullRequest%5C201706230924100930-6%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by gosali** [comment]
@sethvs  Thank you for your feedback and suggestion! 
  
We foundyou have created to brand new articles which is not fully aligned to our acceptance criterial, I am closing this PR. 
  
Microsoft C&E Localization Team

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+Get-AuthenticodeSignature SqlScript.ps1
+```
+
+или:
+
+```
+gas SqlScript.ps1
+```
+
+Если вы создадите псевдоним "word" для программы Microsoft Office Word, вы сможете использовать "word" вместо следюущей команды:
```
Comment (vors) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md]: `следюущей` typo

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+Например:
+
+```
+ac -> Add-Content
+```
+
+Это позволяет получить нужную вам информацию быстро и просто.
+
+Использующий стрелку формат не используется для псевдонимов, включающих дефис.
+Вполне вероятно, что они являются желательными альтернативами именам командлетов и функций, а не просто аббревиатурами или прозвищами, и их автор, возможно, не желал бы, чтобы факт того, что они являются псевдонимами, был настолько прозрачным.
```
Comment (vors) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md]: "прозвищами" doesn't seems like an appropriate term here. :)

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+Для получения дополнительной информации наберите:
+
+```
+Get-Help about_Functions
+```
+
+## ОБЪЕКТЫ ПСЕВДОНИМОВ
+
+Псевдонимы Windows PowerShell представлены объектами, являющимися экземплярами класса System.Management.Automation.AliasInfo. Для получения дополнительной информации об этом типе объектов, читайте статью "AliasInfo Class" в библиотеке Microsoft Developer Network (MSDN) по адресу http:\/\/go.microsoft.com\/fwlink\/?LinkId\=143644.
```
Comment (vors) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md]: Link should be fixed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+Например:
+
+```
+Get-Alias | Get-Member
+```
+
+Для того, чтобы получить значения свойств определенного псевдонима, такого как "dir", получите его объект, затем передайте его по конвейеру командлету Format-List.
+Например, следующая команда получает псевдоним "dir".
+Затем она передает полученный объект по конвейеру командлету Format-List.
+Далее команда использует параметр Property командлета Format-List с символом звездочки (\*) в качестве значения для того, чтобы показать все свойства псевдонима "dir".
```
Comment (vors) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md]: Remove the escaping `\` from `\*`

**Comment by vors** [comment]
> We foundyou have created to brand new articles which is not fully aligned to our acceptance criterial, I am closing this PR.

This is a translation of https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Core/About/about_Aliases.md

**Comment by sethvs** [comment]
Am I get it right: there is no need in Russian version of PowerShell Help?

**Pull request closed without merging**